js/count-letter-in-string.test.js

```diff
@@ -0,0 +1,7 @@
+const count = require('./count-letter-in-string');
+
+describe('Count letter in string', () => {
+  test('If get string aba should return object a:2 b:1', () => {
+    expect(count('aba')).toMatch({ "a": 2, "b": 1 });
```

Comment:
How about omitting the quotes in the object literal properties' names?

```suggestion
    expect(count('aba')).toMatch({ a: 2, b: 1 });
```